src/pages/SignIn/SignIn.js

```diff
@@ -1,7 +1,98 @@
 import React from 'react';
+import styled from 'styled-components';
+// import KakaoLogin from 'react-kakao-login';
 
-function SignIn() {
-  return <div>SignIn</div>;
-}
+const SignIn = () => {
+  return (
+    <Wrapper>
+      <SignUpBox>
+        <Title>로그인</Title>
+        <div>
+          <InputBox placeholder="이메일 주소" type="text" />
+          <InputBox placeholder="비밀번호" type="text" />
+        </div>
+        <div>
+          <Label>
+            <div>
+              <input type="checkbox" />
+              로그인 상태 유지
+            </div>
+          </Label>
+        </div>
+        <ButtonWrapper>
+          <div>
+            <button className="emailBtn">로그인</button>
```

Comment:
```suggestion
            <EmailBtn>로그인</EmailBtn>
```
이렇게 혹은
```suggestion
            <Button type="email">로그인</Button>
```
이렇게 하면 좋을거 같아요! 스타일드 컴포넌트 사용하실 때는 가급적 클래스 셀렉터 사용하지 않는 쪽으로 작성해보세요

Reply:
넵 수정했습니다!